draft-ietf-rats-msg-wrap.md

```diff
@@ -259,7 +259,7 @@ func CMWTypeSniff(b []byte) (CMW, error) {
 Layered attesters and composite devices ({{Sections 3.2 and 3.3 of -rats-arch}}) generate evidence that consists of multiple parts.
 
 For example, in data center servers, it is not uncommon for separate Attesting Environments (AE) to serve a subsection of the entire machine.
-One AE might measure and attest to what was booted on the main CPU, while another AE might measure and attest to what was booted on a SmartNIC plugged into a PCIe slot, and a third AE might measure and attest to what was booted on the machine's GPU.
+One AE might produce Evidence including Claims that represent what was booted on the main CPU, or on a SmartNIC plugged into a PCIe slot, or on the machine's GPU.
```

Comment:
This has changed the meaning.  The *or* seem to present alternatives, but that's not the case: in the composite attester multiple AEs co-exist in parallel.

Reply:
I am going to revert [8c5dcc3](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/45/commits/8c5dcc304cd21a6c6672c3c2ca47fa70f315ee29) and create a new issue where to wordsmith §3.4

Reply:
OK

Reply:
Opened #46 to track this